client/state/data-layer/wpcom/sites/comments-tree/index.js

```diff
@@ -17,15 +17,15 @@ import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { errorNotice } from 'state/notices/actions';
 import { getRawSite } from 'state/sites/selectors';
 
-const fetchCommentsTreeForSite = ( { dispatch }, action ) => {
+export const fetchCommentsTreeForSite = ( { dispatch }, action ) => {
 	const { siteId, status = 'unapproved' } = action.query;
 
 	dispatch(
 		http(
 			{
 				method: 'GET',
 				path: `/sites/${ siteId }/comments-tree`,
-				apiNamespace: 'wpcom/v2',
+				apiVersion: '1',
```

Comment:
As an api namespace? or does apiVersion also accept a leading `v`?

For reference the old endpoint is at wp rest api with the wpcom/v2 namespace:
<img width="910" alt="screen shot 2017-08-19 at 12 32 58 pm" src="https://user-images.githubusercontent.com/1270189/29489640-a861c39e-84da-11e7-9747-d4163c7a0dda.png">

vs wp.com api at version 1

<img width="1037" alt="screen shot 2017-08-19 at 12 32 01 pm" src="https://user-images.githubusercontent.com/1270189/29489642-b0f7ef92-84da-11e7-9302-c45aa75aa1fb.png">